python/src/opendp/context.py

```diff
@@ -403,6 +403,10 @@ def compositor(
 
         if margins:
             for by, margin in margins.items():
+                # wrap single strings in a singleton tuple
+                if not isinstance(by, tuple):
+                    msg = by if isinstance(by, str) else "your-column"
+                    raise ValueError(f"Margin keys must be tuples. For single-valued tuples include a trailing comma, ie: `('{msg}',)`")
```

Comment:
Double check that our coverage checks are working correctly? The ternary introduces a new branch, and I'm a little surprised that that isn't being flagged... but maybe it doesn't count branches within lines?